src/plugins/controls/public/services/data_views/data_views.story.ts

```diff
@@ -17,13 +17,21 @@ let currentDataView: DataView;
 export const injectStorybookDataView = (dataView: DataView) => (currentDataView = dataView);
 
 export const dataViewsServiceFactory: DataViewsServiceFactory = () => ({
-  get: (() =>
-    new Promise((r) =>
-      setTimeout(() => r(currentDataView), 100)
+  get: ((dataViewId) =>
```

Comment:
Updated to reflect actual `dataViews.get` behavior, which is it throws if data view is not found and never returns undefined

Reply:
Nice fix! The more our mocks match the actual behaviour the better!